src/printer-php.js

```diff
@@ -0,0 +1,269 @@
+"use strict";
+
+const docBuilders = require("./doc-builders");
+const concat = docBuilders.concat;
+const join = docBuilders.join;
+const line = docBuilders.line;
+const group = docBuilders.group;
+const indent = docBuilders.indent;
+const hardline = docBuilders.hardline;
+const softline = docBuilders.softline;
+
+function genericPrint(path) {
+  const n = path.getValue();
+  if (!n) {
+    return "";
+  } else if (typeof n === "string") {
+    return n;
+  }
+  return handleNode(n);
+}
+
+function handleLiteral(node) {
```

Comment:
Should these be renamed `print*` to be similar to other printers’ methods?

Reply:
makes sense - updated!